scripts/docker/README

```diff
@@ -0,0 +1,37 @@
+Dockerfiles are used to create Docker images.  Docker images are the 'recipes' used to build virtual environments called containers.
+These dockerfiles all use a base Linux image that has a particular compiler installed.  These base images were created by David
+Beckinsale and are located on Docker Hub at this location:
+https://hub.docker.com/r/axom/compilers/tags
+
+The base images are accessed in these dockerfiles by the use of the 'FROM' command:
+FROM axom/compilers:<compiler>
+
+The remaining dockerfile commands attempt to build the Axom third party libraries using the specified compiler.
+The following commands update the compiler and add Fortran compiler support:
+RUN sudo apt-get update -y
+RUN sudo apt-get install curl -fy
+RUN sudo apt-get -qq install -y --no-install-recommends gfortran-7 && sudo update-alternatives --install /usr/bin/gfortran gfortran /usr/bin/gfortran-7 100
+#(For Clang Builds - Need to restore stdc++ lib)
+RUN sudo apt-get install libstdc++-7-dev
+
+Then the Axom and RAJA source is retrieved and RAJA is built and installed: (WORKDIR functions as a 'cd' command)
+RUN git clone --recursive https://github.com/LLNL/axom.git
+RUN git clone --recursive https://github.com/LLNL/RAJA.git
+WORKDIR "/home/axom/RAJA"
+RUN mkdir build
+WORKDIR "/home/axom/RAJA/build"
+RUN cmake ../. -DENABLE_TESTS=OFF
+RUN make 
+RUN sudo make install
+
+Finally, the uberenv script is invoked to build the libraries and store them in the axom_tpls directory.  (The host config files
+produced during this process had been added to the Axom repo under "axom/host-configs/docker")
+WORKDIR "/home/axom/axom"
+RUN ./scripts/uberenv/uberenv.py --spack-config-dir=./scripts/uberenv/spack_configs/docker --spec=%clang@4.0.0^mpich@3.0.4 --prefix=/home/axom/axom_tpls -k
```

Comment:
@keithhealy -- can you please add a line about requiring ``mpich`` for docker.  This will document the trouble you had with the default spack MPI.